cmake/SetupCompilerOptions.cmake

```diff
@@ -282,6 +282,8 @@ endif()
 SET( CMAKE_CXX_EXTENSIONS OFF )
 SET( CMAKE_CXX_STANDARD_REQUIRED ON )
 
+if (BLT_CXX_STD)
+
 if( BLT_CXX_STD STREQUAL c++98 ) 
```

Comment:
Is this indented?

Reply:
No, but it should be 😄

Reply:
Approved after you fix your heinous crime against code styling... thanks for fixing this!

Reply:
No problems. I should use tabs right? 😂

Reply:
Please mix tabs and spaces...